docs/machine-learning/resources/transforms.md

```diff
@@ -1,168 +1,102 @@
-| <xref:Microsoft.ML.Transforms.Normalizers.MeanVarDblAggregator> | Computes the mean and variance for a vector valued column. It tracks the current mean and the M2 (sum of squared diffs of the values from the mean), the number of NaNs and the number of non-zero elements. |
-| <xref:Microsoft.ML.Transforms.Normalizers.MeanVarSngAggregator> | Computes the mean and variance for a vector valued column. It tracks the current mean and the M2 (sum of squared diffs of the values from the mean), the number of NaNs and the number of non-zero elements. |
-| <xref:Microsoft.ML.Transforms.Normalizers.MinMaxDblAggregator> | Tracks min, max, number of non-sparse values (vCount) and number of ProcessValue() calls (trainCount) for a vector valued column. |
-| <xref:Microsoft.ML.Transforms.Normalizers.NormalizeTransform> | Standardizes feature ranges. |
-| <xref:Microsoft.ML.Transforms.Normalizers.NormalizingTransformer> |Standardizes feature ranges. |
+| <xref:Microsoft.ML.TextCatalog.FeaturizeText*> | Transform a text column into a float array of normalized ngrams and char-grams counts | 
+| <xref:Microsoft.ML.TextCatalog.TokenizeIntoWords*> | Split one or more text columns into individual words |
+| <xref:Microsoft.ML.TextCatalog.TokenizeIntoCharactersAsKeys*> | Split one or more text columns into individual characters floats over a set of topics |
+| <xref:Microsoft.ML.TextCatalog.NormalizeText*> | Change case, remove diacritical marks, punctuation marks and numbers |
+| <xref:Microsoft.ML.TextCatalog.ProduceNgrams*> | Transform text column into a bag of counts of ngrams (sequences of consecutive words)|
+| <xref:Microsoft.ML.TextCatalog.ProduceWordBags*> | Transform text column into a bag of counts of ngrams vector |
+| <xref:Microsoft.ML.TextCatalog.ProduceHashedNgrams*> | Transform text column into a vector of hashed ngram counts |
+| <xref:Microsoft.ML.TextCatalog.ProduceHashedWordBags*> | Transform text column into a bag of hashed ngram counts |
+| <xref:Microsoft.ML.TextCatalog.RemoveDefaultStopWords*>  | Remove default stop words for the specified language from input columns |
+| <xref:Microsoft.ML.TextCatalog.RemoveStopWords*> | Removes specified stop words from input columns |
+| <xref:Microsoft.ML.TextCatalog.LatentDirichletAllocation*> | Transform a document (represented as a vector of floats) into a vector of floats over a set of topics |
+| <xref:Microsoft.ML.TextCatalog.ApplyWordEmbedding*> | Convert vectors of text tokens into sentence vectors using a pre-trained model |
 
-## Onnx
+## Image transformations
 
 | Transform | Definition |
 | --- | --- |
-| <xref:Microsoft.ML.Transforms.OnnxTransform> | Scores pre-trained ONNX models which use the ONNX standard v1.2 |
+| <xref:Microsoft.ML.ImageEstimatorsCatalog.ConvertToGrayscale*> | Convert an image to grayscale |
+| <xref:Microsoft.ML.ImageEstimatorsCatalog.ConvertToImage*> | Convert a vector of pixels to ImageType |
```

Comment:
does this need a reference to the .NET ImageType?

Reply:
Sure

Reply:
Just checking that it is actually this type Microsoft.ML.Transforms.Image.ImageDataViewType?